SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -0,0 +1,229 @@
+# This spec file assumes you're building on an environment where:
+# * You have access to the internet during the build
+# * You have rustup installed on your system
+# * You have both x86_64-unknown-linux-gnu and x86_64-unknown-linux-musl, or
+# * You have both aarch64-unknown-linux-gnu and aarch64-unknown-linux-musl
+#   targets installed.
+
+%define using_rustup 0
+%define using_musl_libc 0
+%define using_vendored_crates 1
+
+# Generated by update script
+%global gitver   38.0
+%global gitnum   72
+%global gitrev   8f7209bdc
+%global clh_name cloud-hypervisor
```

Comment:
We can get rid of those two variables if we can rename our tar files to avoid ambiguity with the proper CLH tar files

Reply:
Would like to take this a step forward and remove gitrev from release number altogether. We can find that info by looking at the version used in https://github.com/microsoft/cloud-hypervisor/tags eg

<img width="315" alt="image" src="https://github.com/microsoft/azurelinux/assets/30801614/bdab63e8-93bc-4d7c-9496-5d5c91b533c0">

This would also further align with normal clh